release/tasks/copy-template/README.md

```diff
@@ -12,6 +12,7 @@ A bundle of predefined templates to use can be found in [Common Templates](https
 - **targetTemplateName**: Name of an target OpenShift template.
 - **targetTemplateNamespace**: Namespace of an target OpenShift template to create in. (defaults to active namespace)
 - **allowReplace**: Allow replacing already existing template (same combination name/namespace). Allowed values true/false
+- **setOwnerreference**: Set owner reference to the new object created by the task run pod. Allowed values true/false
```

Comment:
CamelCase? `setOwnerReference`?

Reply:
I am using same naming convention as kubectl task https://hub.tekton.dev/tekton/task/kubectl-deploy-pod

Reply:
`set-ownerreference` is yet another variant? They also called it `ownerReferences` in their docs. To me `setOwnerReference` looks correct.

Reply:
updated